entry.rb

```diff
@@ -11,6 +11,10 @@
 extra_words_files = ARGV[1].shellsplit.flat_map { |path| Dir.glob(path) }
 args = ARGV[2].shellsplit
 
+args << '--add-tex-command=problemname P'
+args << '--add-tex-command=href pP'
+args << '--add-tex-command=illustration ppP'
+
 puts ARGV
```

Comment:
```suggestion
puts args
```

Probably want to print args as we want the full argument list not just what the user sends in.